c7n/filters/core.py

```diff
@@ -349,6 +350,11 @@ def process_value_type(self, sentinel, value):
                     value = parse(value)
                 except (AttributeError, TypeError):
                     value = 0
+
+            # Work around placebo issues with tz
+            if isinstance(value, datetime) and not value.tzinfo:
+                value = pytz.utc.localize(value)
```

Comment:
i'd say just use value.replace(tzinfo=tzutc()) although i'm missing exact context for the comparison (tzutc().fromutc(dt) will convert to local time), might be worth linking the upstream pull request https://github.com/garnaat/placebo/pull/50